src/RuleSet/Sets/PhpCsFixerSet.php

```diff
@@ -104,6 +104,7 @@ public function getRules(): array
                 'only_booleans' => true,
             ],
             'ordered_class_elements' => true,
+            'phpdoc_to_comment' => false,
```

Comment:
Disabled to prevent an unwanted change to the comment added in [`tests/Test/AbstractIntegrationCaseFactory.php`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6680/files#diff-10c2773925f6824fcb19f0e6ec90d2f1c271970f6fcc615e0f9103ccf28ff132R178).